src/functions/Growth.java

```diff
@@ -0,0 +1,14 @@
+package functions;
+
+import Utils;
+import functions.IFunction;
+
+public class Growth implements IFunction {
```

Comment:
Unique functionality can only be defined once per IFunction implementation, i.e: if Leaf and Fruit have different Growth functionality, need to define as LeafGrowth and FruitGrowth -- not general functionality. This allows for developers to define both general and specific functionality, then assign them in config files

This is may be better than defining all types in a single Growth function; no need for inheritance as there is currently little to inherent (reconsider if fields need to be inherited)

Reply:
Reduce number of inheriting structures, can still allow inheritance if needed